apps/silverback-drupal/web/modules/custom/silverback_gatsby_test/src/Entity/Server.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Drupal\silverback_gatsby_test\Entity;
+
+use Drupal\Core\Utility\Error;
+use Drupal\graphql\Entity\Server as ServerOriginal;
+use Drupal\graphql\GraphQL\Execution\ExecutionResult as CacheableExecutionResult;
+use GraphQL\Server\OperationParams;
+
+// TODO: Remove once https://github.com/drupal-graphql/graphql/issues/1195 is
```

Comment:
Shouldn't we create a patch and a proxy package instead?

Reply:
Yeah, it's my plan to submit a proper patch to https://github.com/drupal-graphql/graphql. Just need time 😬